package.json

```diff
@@ -100,7 +100,7 @@
     "react-native-listener": "1.0.1",
     "react-phone-number-input": "^0.15.2",
     "react-redux": "4.4.6",
-    "react-webcam-onfido": "0.0.19",
+    "react-webcam-onfido": "0.0.20",
```

Comment:
It's the first time I see this module. Is it a fork?

Reply:
Yeah. you can see the related changes here: https://github.com/onfido/react-webcam/pull/10